typings/storybook__addons/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions extension for @storybook/addons 6.0.18
+// Project: https://github.com/storybookjs/storybook/tree/next/lib/addons
+// Definitions by: Martin Hochel
+// Definitions: N/A
+// TypeScript Version: 3.1
+
+import { Parameters } from '@storybook/addons';
+
+declare module '@storybook/addons' {
+  // PUBLIC API - extended definitions
+  export interface Parameters extends ParametersExtended {}
+
+  // =====
+  interface ParametersExtended {
+    controls?: ControlsParameters & DisableControl;
+  }
+
+  interface ControlsParameters {
```

Comment:
while this exists in SB github repo it is not being exposed for consumers (also most of these are completely missing in 6.0.x so probably might not work - sort doesn't work (tried that out), rest seems to be working )